index.js

```diff
@@ -76,6 +96,51 @@ const csslint = function(target, config, dest) {
 
 };
 
+const csslintFix = function(target, config, dest) {
+  var args = _.assign(stylefmtDefaults, config);
+
+  const modifier = function(css) {
+    var openingWhitespace = css.match(/^[\r\n\t ]+/);
+    var trailingWhitespace = css.match(/[\t ]+$/);
+    var indentation;
+    var output;
+
+    if (openingWhitespace) {
+      indentation = openingWhitespace[0].match(/^([\t ]+)/m);
```

Comment:
Can indentation be a mix of tabs and spaces? Does this mean the step in 123 reincorporates those mixed tabs and spaces?